{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "5c9eea62_5feccf9c",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 10366
      },
      "writtenOn": "2024-03-12T12:19:10Z",
      "side": 1,
      "message": "recheck\n\nUnrelated py311 failure",
      "revId": "496d48a2372e30305940a7bd0020aceb92391a0c",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543"
    }
  ]
}